### PR TITLE
add usuful link

### PR DESCRIPTION
**Pull request by Futsum1**
## Purpose
updated pull request and provided more resource 

## Approach 
I added more documents, DevOps resource links, and Git cheat sheet  to the README.md 

## Learning
hands-on experience with a fundamental aspect of collaboration on Github pull request

Closes #6

### CODE REVIEW

**Review by fedluk70 — COMMENTED** [review]
good job F

**Comment by Futsum1** [comment]
*------*

**Comment by fedluk70** [comment]
nice job

**Review by anjuyeddu — COMMENTED** [review]
great Github resources

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 
 
+##
```
Comment (michaelachrisco) [inline README.md]: What is this line doing? Do we need this line in the markdown?

File: README.md [hunk 2]
```diff
@@ -12,6 +13,17 @@
 
 *All combined notes, links and documents are listed below:
 
+mac-6-add-sp-link
```
Comment (michaelachrisco) [inline README.md]: Lets remove the branch names within the documentation.

File: README.md [hunk 3]
```diff
@@ -71,4 +83,8 @@
   
 
 
+mac-6-add-sp-link
+#### PULL REQUESTS FOR BRAGGING RIGHTS 
+
 
+ development
```
Comment (michaelachrisco) [inline README.md]: Same, lets remove the branch name from the documentation.

File: README.md [hunk 4]
```diff
@@ -71,4 +83,8 @@
   
 
 
+mac-6-add-sp-link
```
Comment (michaelachrisco) [inline README.md]: Same here. Lets remove the branch name from the pull request.

File: README.md [hunk 5]
```diff
@@ -12,6 +13,17 @@
 
 *All combined notes, links and documents are listed below:
 
+mac-6-add-sp-link
+#### [Github Documentation](https://docs.github.com/en/github)
+### [Git and Github tutorials on Codecademy](https://www.codecademy.com/learn/learn-git)
```
Comment (michaelachrisco) [inline README.md]: Why does this link have a different header than the other links?

File: README.md [hunk 6]
```diff
@@ -12,6 +13,17 @@
 
 *All combined notes, links and documents are listed below:
 
+mac-6-add-sp-link
+#### [Github Documentation](https://docs.github.com/en/github)
```
Comment (michaelachrisco) [inline README.md:29]: I really like the inclusion of the original docs!

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
Awesome job on your first PR @Futsum1! I looked at the links to make sure they do not 404 and they all are really good resources. I particularly like the inclusion of the `git` resources. Nice!

Couple of blockers: 
1. Lets remove Headers markdown IE `##` that does not contain a header. 
2. Lets remove branch names from the documentation. The branch names are not useful to know outside the pull request.

File: README.md (hunk 5 above)
Comment (Futsum1) [reply README.md]: Hi Michael, That happened by mistake, I just fixed. All has to be the same. Thanks for that

File: README.md (hunk 1 above)
Comment (Futsum1) [reply README.md]: Not at all, The branch name shouldn't be at the "Read me" I'm not sure how i added it there, it was probably by mistake.

Comment (michaelachrisco) [reply README.md]: All good, thanks for removing those references! Maybe we had a merge conflict at some point? Either way, looks good now. Thanks for taking care of that.

File: README.md [hunk 7]
```diff
@@ -13,6 +15,14 @@
...
 *All combined notes, links and documents are listed below:
 
+#### [Github Documentation](https://docs.github.com/en/github)
+#### [Git and Github tutorials on Codecademy](https://www.codecademy.com/learn/learn-git)
+#### [Git and Github tutorials on Udemy](https://www.udemy.com/topic/git/)
+#### [Pro Git book by Scott Chacon and Ben Straub](https://git-scm.com/book/en/v2)
+#### [Youtube tutorials on Git and Github](https://www.youtube.com/results?search_query=git+github+tutorial.)
+
+
+## This is my fourth PR
```
Comment (michaelachrisco) [inline README.md]: I believe this line is the last thing that needs to be removed and we should be good!

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
Thanks for making the requested changes @Futsum1 ! Looks good. 

One last blocker, looks like there is one last line that needs to be removed, and we should be good.

**Review by fedluk70 — COMMENTED** [review]
Looks good

File: README.md (hunk 7 above)
Comment (Futsum1) [reply README.md]: Thanks, I removed the last line.

File: README.md (hunk 6 above)
Comment (Ruhama21) [reply README.md:29]: _Thank you for your help. michaelchrisco!_

**Review by Ruhama21 — COMMENTED** [review]
Thank you for letting me to review changes! please Add more link!

**Review by michaelachrisco — APPROVED** [review]
Woo!! Everything looks good, thanks for everything @Futsum1 !

**Comment by Futsum1** [comment]
added more links

**Pull request merged**